lib/Cake/Test/Case/Utility/DebuggerTest.php

```diff
@@ -156,6 +156,25 @@ public function testOutput() {
 	}
 
 /**
+ * test encodes error messages
+ *
+ * @return void
+ */
+	public function testOutputEncodeDescription() {
+		$this->skipIf(version_compare(PHP_VERSION, '5.4.0', '<'), 'Index errors are different in PHP 5.3');
+		set_error_handler('Debugger::showError');
+		$this->_restoreError = true;
+
+		ob_start();
+		$a = 'things';
```

Comment:
Why is this a string? If this is an empty array, I think the test would pass even in PHP5.3.